app/assets/stylesheets/quotes.scss

```diff
@@ -0,0 +1,3 @@
+.quote-example span, q {
```

Comment:
this likely doesn't do what you expect it to. This is equivalent to:

```
.quote-example span {
  font-style: italic;
}
q {
  font-style: italic;
}
```
You probably intended for: 
```
.quote-example span, .quote-example q {
  font-style: italic;
}
```
Or because we're writing sass, we can do:

```

.quote-example {
  span, q {
    font-style: italic;
  }
}
```
Which removes the need to duplicate the class.